tests/playwright/sanity/container-grid.test.ts

```diff
@@ -700,6 +700,15 @@ test.describe( 'Container Grid tests @container', () => {
 			expect.soft( linkElement ).toHaveAttribute( 'href', 'https://go.elementor.com/widget-container-grid' );
 		} );
 
+		await test.step( 'Select existing grid container', async () => {
```

Comment:
Do we want to check what happens if we change from flex to grid?

Reply:
It is tested on line 672.
This test is to make sure that the link is rendered correctly after selecting other elements